korangar/src/graphics/passes/point_shadow/shader/entity_bindless.wgsl

```diff
@@ -121,6 +138,50 @@ fn vertex_data(vertex_index: u32) -> Vertex {
     return Vertex(vec3<f32>(x, y, z), vec2<f32>(u, v), depth, curve);
 }
 
+fn vertex_data_new(vertex_index: u32, instance_index: u32) -> Vertex {
```

Comment:
Uff, non-uniform branching is not good. Can you try to make this branchless? (By using a truth table and maybe by using the "select" function?) You can also remove the old vertex data function then.

Reply:
I want to try to shift using matrix, but my calculation abilities is limited...

Reply:
It's fine for now. But could you please create a table in the documentation like we did with the other functions? That would help greatly if we want to debug this part later!